ansible/update_pw_python.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  tasks:
+    - include: update_pw_python.yml
+      become: true
+      become_user: root
+- name: Fetch updated piecewise repo
+  git: repo=https://github.com/opentechinstitute/piecewise.git 
+       dest=/opt/piecewise.git/
+       version=update-ndt-test
+- name: Deploy updated piecewise code
+  file: src=/opt/piecewise.git/{{ item.src }} dest=/opt/{{ item.dest }} state=link
```

Comment:
Since this is Ansible code from the original application and it's not technically incorrect, I'd like to defer dealing with this in commits addressing Issue #98 specifically.